src/main/java/md2k/mCerebrum/cStress/features/ECGFeatures.java

```diff
@@ -385,11 +383,12 @@ private void filterPeaksTemp2(DataPointStream Rpeak_temp2, DataPointStream Rpeak
             }
 
             for (int j = 0; j < diffRpeak.size(); j++) {
-                if (diffRpeak.get(j).timestamp < (AUTOSENSE.RPEAK_INTERPEAK_MULTIPLIER * frequency)) {
+                if (diffRpeak.get(j) < (AUTOSENSE.RPEAK_INTERPEAK_MULTIPLIER*1000)) {
```

Comment:
The matlab code uses indices.  The java code use timestamps.  It's the same condition of whether the two peaks are within half a second.